### DISCUSSION TITLE
[Windows & Mac] Stage Display & multiple outputs

### DISCUSSION THREAD

**Discussion by pysche02** [ChurchApps/ChurchAppsSupport#316] [FreeShow] [answered]
How can I make it work? I tried to add an input through the settings but it keeps on showing the main display. I can't seem to figure out how can I make the stage display. Does FreeShow have multiple outputs where 1 screen is for main display 1 screen for stage display and 1 screen is for lower thirds?

**Comment by vassbo** [comment]
Hi, to move output windows to another screen, please select "Choose screen" in the output settings. https://freeshow.app/docs/output
You can add many different output windows and change the styling to use for different things, like main output, lower third text, etc. https://freeshow.app/docs/styles
Stage displays are seperate and can be accessed from any web browser. Go to the "Connection" settings tab to find out more. https://freeshow.app/docs/stage

**Reply by pysche02** [reply]
I tried to input the link that was shows for stage display however it does not work. Does it require a pre-requisite app?

**Reply by vassbo** [reply, marked as answer]
No other app required. Try clicking on the link in the program, and see if it works on the same computer the program it's installed on, then try the same exact link on another device connected to the same router.

It should always work on the same computer with the program, but if it doesn't work on another device, connected to the same router, then you have to allow it in the router settings.

More here: https://freeshow.app/docs/connecting#troubleshooting
